image/templates/files/compliance-cpu-loadgen.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/ksh93
+#
+# Copyright 2025 Oxide Computer Company
+#
+
+if (( $# != 1 )); then
+	echo "usage: compliance-cpu-loadgen.sh <numthreads>" >&2
+	exit 1
+fi
+
+integer nthreads=$1
+
+if (( nthreads == 0 )); then
+	nthreads=$(psrinfo -t)
+fi
+
+for i in {1..$nthreads}; do
+	while true; do :; done &
+done
+
+exit 0
```

Comment:
Just a suggestion to keep with the framework (although the result is exactly the same)
```suggestion
exit $SMF_EXIT_OK
```
Along with sourcing `smf_include.sh` at the top.

Reply:
Done.